<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>main_window</class>
 <widget class="QMainWindow" name="main_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>800</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string notr="true">Yin &amp; Yang</string>
  </property>
  <property name="windowIcon">
   <iconset theme="sh.oskar.yin_yang" resource="../resources/resources.qrc">
    <normaloff>:/icons/icon</normaloff>:/icons/icon</iconset>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedKingdom"/>
  </property>
  <widget class="QWidget" name="central_widget">
   <layout class="QVBoxLayout" name="central_widget_layout">
    <item>
     <widget class="QTabWidget" name="tab_widget">
      <property name="minimumSize">
       <size>
        <width>368</width>
        <height>0</height>
       </size>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="settings">
       <attribute name="title">
        <string>Settings</string>
       </attribute>
       <layout class="QVBoxLayout" name="settings_layout">
        <item>
         <widget class="QCheckBox" name="btn_enable">
          <property name="text">
           <string>Automatic theme switching</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="schedule_settings" native="true">
          <layout class="QVBoxLayout" name="schedule_settings_layout">
           <item>
            <widget class="Line" name="line_top">
             <property name="orientation">
              <enum>Qt::Orientation::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="btn_schedule">
             <property name="text">
              <string>Custom Schedule</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QFrame" name="time">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <layout class="QFormLayout" name="time_layout">
              <property name="labelAlignment">
               <set>Qt::AlignmentFlag::AlignCenter</set>
              </property>
              <property name="leftMargin">
               <number>37</number>
              </property>
              <item row="0" column="0">
               <widget class="QLabel" name="label_light">
                <property name="text">
                 <string>Light:</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QTimeEdit" name="inp_time_light">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>88</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="displayFormat">
                 <string notr="true">HH:mm</string>
                </property>
                <property name="time">
                 <time>
                  <hour>8</hour>
                  <minute>0</minute>
                  <second>0</second>
                 </time>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="label_dark">
                <property name="text">
                 <string>Dark:</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QTimeEdit" name="inp_time_dark">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>88</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="displayFormat">
                 <string notr="true">HH:mm</string>
                </property>
                <property name="time">
                 <time>
                  <hour>20</hour>
                  <minute>0</minute>
                  <second>0</second>
                 </time>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="btn_sun">
             <property name="text">
              <string>Sunset to Sunrise</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QFrame" name="location">
             <layout class="QVBoxLayout" name="verticalLayout">
              <item>
               <widget class="QWidget" name="location_input" native="true">
                <layout class="QFormLayout" name="formLayout">
                 <item row="1" column="0">
                  <widget class="QLabel" name="label_longitude">
                   <property name="text">
                    <string>Longitude:</string>
                   </property>
                   <property name="alignment">
                    <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
                   </property>
                  </widget>
                 </item>
                 <item row="1" column="1">
                  <widget class="QDoubleSpinBox" name="inp_longitude">
                   <property name="sizePolicy">
                    <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                     <horstretch>0</horstretch>
                     <verstretch>0</verstretch>
                    </sizepolicy>
                   </property>
                   <property name="minimumSize">
                    <size>
                     <width>88</width>
                     <height>0</height>
                    </size>
                   </property>
                   <property name="suffix">
                    <string notr="true">°</string>
                   </property>
                   <property name="minimum">
                    <double>-180.000000000000000</double>
                   </property>
                   <property name="maximum">
                    <double>180.000000000000000</double>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="0">
                  <widget class="QLabel" name="label_latitude">
                   <property name="text">
                    <string>Latitude:</string>
                   </property>
                   <property name="alignment">
                    <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="1">
                  <widget class="QDoubleSpinBox" name="inp_latitude">
                   <property name="sizePolicy">
                    <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                     <horstretch>0</horstretch>
                     <verstretch>0</verstretch>
                    </sizepolicy>
                   </property>
                   <property name="minimumSize">
                    <size>
                     <width>88</width>
                     <height>0</height>
                    </size>
                   </property>
                   <property name="suffix">
                    <string notr="true">°</string>
                   </property>
                   <property name="minimum">
                    <double>-90.000000000000000</double>
                   </property>
                   <property name="maximum">
                    <double>90.000000000000000</double>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="btn_location">
                <property name="text">
                 <string>update location automatically</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="Line" name="line_bottom">
             <property name="orientation">
              <enum>Qt::Orientation::Horizontal</enum>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="manual_buttons" native="true">
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QPushButton" name="button_light">
             <property name="text">
              <string>Light</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="button_dark">
             <property name="text">
              <string>Dark</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="toggle_notification">
          <property name="text">
           <string>Send a notification</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QFormLayout" name="bootOffsetSettings">
          <item row="0" column="0">
           <widget class="QLabel" name="bootOffsetLabel">
            <property name="toolTip">
             <string>Time to wait until the system finished booting. Default value is 10 seconds.</string>
            </property>
            <property name="text">
             <string>Delay after boot:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QSpinBox" name="bootOffset">
            <property name="suffix">
             <string>s</string>
            </property>
            <property name="value">
             <number>10</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="label_active">
          <property name="text">
           <string notr="true">Darkmode will be active between</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="space_vertical">
          <property name="orientation">
           <enum>Qt::Orientation::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="plugins">
       <attribute name="title">
        <string>Plugins</string>
       </attribute>
       <layout class="QVBoxLayout" name="plugins_layout">
        <item>
         <widget class="QScrollArea" name="plugins_scroll">
          <property name="frameShape">
           <enum>QFrame::Shape::NoFrame</enum>
          </property>
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="plugins_scroll_content">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>525</width>
             <height>657</height>
            </rect>
           </property>
           <layout class="QVBoxLayout" name="plugins_scroll_content_layout">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout">
              <item>
               <widget class="QGroupBox" name="samplePluginGroupBox">
                <property name="title">
                 <string notr="true">Sample Plugin</string>
                </property>
                <layout class="QHBoxLayout" name="horizontalLayout_2">
                 <item>
                  <widget class="QComboBox" name="comboBox">
                   <property name="currentText">
                    <string notr="true"/>
                   </property>
                   <property name="sizeAdjustPolicy">
                    <enum>QComboBox::SizeAdjustPolicy::AdjustToContentsOnFirstShow</enum>
                   </property>
                   <property name="placeholderText">
                    <string notr="true">firefox-compact-light@mozilla.org</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QComboBox" name="comboBox_2">
                   <property name="currentText">
                    <string notr="true"/>
                   </property>
                   <property name="sizeAdjustPolicy">
                    <enum>QComboBox::SizeAdjustPolicy::AdjustToContentsOnFirstShow</enum>
                   </property>
                   <property name="placeholderText">
                    <string notr="true">firefox-compact-dark@mozilla.org</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QDialogButtonBox" name="btn_box">
      <property name="standardButtons">
       <set>QDialogButtonBox::StandardButton::Apply|QDialogButtonBox::StandardButton::Cancel|QDialogButtonBox::StandardButton::RestoreDefaults</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="status_bar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../resources/resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>btn_sun</sender>
   <signal>toggled(bool)</signal>
   <receiver>location</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>289</x>
     <y>225</y>
    </hint>
    <hint type="destinationlabel">
     <x>270</x>
     <y>353</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_schedule</sender>
   <signal>toggled(bool)</signal>
   <receiver>time</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>130</x>
     <y>109</y>
    </hint>
    <hint type="destinationlabel">
     <x>235</x>
     <y>197</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_enable</sender>
   <signal>toggled(bool)</signal>
   <receiver>schedule_settings</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>198</x>
     <y>66</y>
    </hint>
    <hint type="destinationlabel">
     <x>188</x>
     <y>318</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_enable</sender>
   <signal>toggled(bool)</signal>
   <receiver>manual_buttons</receiver>
   <slot>setHidden(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>109</x>
     <y>58</y>
    </hint>
    <hint type="destinationlabel">
     <x>274</x>
     <y>398</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
